### PR TITLE
Add services SergeyShalyapin

### PR DESCRIPTION
**Pull request by Eberli**

### CODE REVIEW

File: src/components/service/ServiceMappings.js [hunk 1]
```diff
@@ -28,13 +33,18 @@ export default class SampleServices {
         this.serviceOrgIDToComponent[this.generateUniqueID("snet", "s2vt-video-captioning")] = S2VTVideoCaptioning;
         this.serviceOrgIDToComponent[this.generateUniqueID("snet", "yolov3-object-detection")] = YOLOv3ObjectDetection;
         this.serviceOrgIDToComponent[this.generateUniqueID("snet", "zeta36-chess-alpha-zero")] = Zeta36ChessAlphaZero;
```
Comment (raamb) [inline src/components/service/ServiceMappings.js:37]: On the mainnet these services will be registered under the snet org correct? We should have the same for Kovan and Ropsten as well

Comment (Eberli) [reply src/components/service/ServiceMappings.js:37]: These services are currently in master snet-dapp and do not belong to my area of responsibility. I add services of the organization SergeyShalyapin. Our services are available on the kovan.

Comment (raamb) [reply src/components/service/ServiceMappings.js:37]: Looks like my comment got the wrong code block. I was referring to the new services under the SergeyShalyapin organization. All services from Singularitynet should be under the snet organization to keep things consistent across Kovan and Ropsten. Can you please republish these services under snet org and change the org name to snet

Comment (arturgontijo) [reply src/components/service/ServiceMappings.js:37]: @Eberli I can add you as a `snet` organization member on Kovan and Ropsten.
As a member you'll be able to publish your services under `snet`.
Is `0xdFdAbd40b28D1aA137b5320fe0E260A3F62268fD` the address that you want to be added?
if you want to use another address, let me know on Slack.

Comment (Eberli) [reply src/components/service/ServiceMappings.js:37]: Thank's. We will republish our services under the snet organization on February 11 and update PR. 
The address is correct.

**Pull request merged**